### ISSUE TITLE
Guest Entries Plugin Validation Errors Missing

### ISSUE THREAD

**Issue by philipboomy** [putyourlightson/craft-sprig#94]
### Question
The Guest Entry Plugin https://github.com/craftcms/guest-entries#usage  returns with an entry variable if there validation errors but that is not returned when using Sprig. I changed it to something unique "guestentry" and then I did this

```
{% if sprig.isInclude %}
{% else %}
    {{ dump(guestentry) }}
{% endif %}
```

but I get "variable **guestentry** is not defined" error. If I dont try to dump that variable after submitting I am getting a response 200 like the form submitted fine but of course not with success.

**Comment by philipboomy** [comment]
For now I decided to use ajax to submit ( I know we are suppose to avoid that :) ) as it gives me a success message too. The guest entries plugin does not return a success variable.

**Comment by bencroker** [comment]
As of Sprig 1.3.0, a variable called `entry` should be returned on error and a boolean variable called `success` should be returned regardless of whether there were errors or not. Is this not the case?

**Comment by philipboomy** [comment]
I am getting a success boolean back but not the entry variable which I have renamed to guestentry. I have tested with both and dumping it but I get that error above about not being defined.

**Comment by bencroker** [comment]
It looks like a variable called `variables` is returned, containing the entry...
https://github.com/craftcms/guest-entries/blob/c69b1f90f6f675f43336dcb5f0970b30173779d7/src/controllers/SaveController.php#L205-L208

Can you try this?

```twig
{{ dump(variables.guestentry) }}
```

**Comment by bencroker** [comment]
I just fixed this in https://github.com/putyourlightson/craft-sprig/commit/921e9bae27b0f6de9048584b2c12900568147545 for the next release, so that the variable `guestentry` will be available in the template.

**Comment by philipboomy** [comment]
Yes {{ dump(variables.guestentry) }} works - Cool thanks for looking into it and fixing so quick!

**Comment by bencroker** [comment]
Released in version 1.3.3.
